examples/react-uploader-adapter/src/index.css

```
@@ -1,31 +1,32 @@
 body {
   margin: 0;
   padding: 0 24px 24px;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
-    sans-serif;
+  font-family:
+    -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell",
```

Stylelint (17.9.0)

[error] Expected no quotes around "Roboto" (font-family-name-quotes)

(font-family-name-quotes)